ergo-lib/src/wallet/box_selector.rs

```diff
@@ -58,6 +62,10 @@ pub enum BoxSelectorError {
     /// Token amount err
     #[error("TokenAmountError: {0:?}")]
     TokenAmountError(#[from] TokenAmountError),
+
+    /// Boxes out of bounds
+    #[error("Boxes is out of bounds")]
+    OutOfBounds(BoundedVecOutOfBounds),
```

Comment:
```suggestion
    OutOfBounds(#[from] BoundedVecOutOfBounds),
```

should remove the need for manual `From` impl below.